cli/types/tests/actions.ts

```diff
@@ -87,3 +87,17 @@ namespace CypressActionCommandOptionTests {
   cy.get('el').click({scrollBehavior: false})
   cy.get('el').click({scrollBehavior: true}) // $ExpectError
 }
+
+namespace CyEventEmitterTests {
```

Comment:
Can we put a comment here (or alongside the types themselves) explaining why we'd expect errors for `waitFor`/`prependListener`? Without context, these assertions seem a little random.

Reply:
I wrote the reason like below:

```js
// `waitFor` doesn't exist in Node EventEmitter 
// and it confuses the users with `cy.wait`
```